AIDevGallery/Samples/SharedCode/SemanticComboBox.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using AIDevGallery.Telemetry.Events;
```

Comment:
should we also exclude-line these usings?

Reply:
I don't think we have to, since the exported projects are building fine (tests are passing). I believe these are already being removed from the SharedCode source generator.

Reply:
they show up in sample code 
<img width="707" alt="image" src="https://github.com/user-attachments/assets/88b49ef0-5ece-4a95-924b-966552f7baf8" />

Reply:
Oh, interesting. This doesn't show up in the exported project itself. This is probably being filtered at the wrong moment. It needs to happen on the SourceGenerator, so during compilation, not project export. I'll fix this.

Reply:
Done.